src/world/mod.rs

```diff
@@ -1,16 +1,99 @@
 pub mod asset;
 pub mod components;
-pub mod json;
 
 /// `bevy_ecs_tiled` world related public exports
 pub mod prelude {
     pub use super::asset::*;
     pub use super::components::*;
+    pub use super::WorldChunkedMaps;
 }
 
 use crate::prelude::*;
-use bevy::{asset::RecursiveDependencyLoadState, prelude::*};
-use bevy_ecs_tilemap::prelude::*;
+use bevy::prelude::*;
+
+#[derive(Resource, Default)]
+pub struct WorldChunkedMaps(pub Vec<(Entity, Rect)>);
+
+pub(crate) fn world_chunking(
+    camera: Query<&Transform, (With<Camera>, Changed<Transform>)>,    
+    worlds: Res<Assets<TiledWorld>>,
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut world_query: Query<(Entity, &TiledWorldHandle, &TiledWorldSettings)>,
+    mut chunked_maps: ResMut<WorldChunkedMaps>,
+) {
+    let camera_transform = match camera.iter().next() {
```

Comment:
What if we have several cameras ?
I think we should iterate over all of them, so we can construct a list of "maps to remove" and a list of "maps to add" then do the actually spawn / despawn.

Reply:
Added handling for multiple cameras.